src/domain/cards/berthDetails/BerthDetails.tsx

```diff
@@ -42,36 +45,36 @@ const BerthDetails: React.SFC<BerthDetailsProps> = ({
 }) => {
   const { t, i18n } = useTranslation();
 
-  const leasesElements = leases.map(({ startDate, endDate, customer }, i) => {
-    return (
-      <div key={i}>
-        <Text>{`${formatDate(startDate, i18n.language)} - ${formatDate(
-          endDate,
-          i18n.language
-        )}`}</Text>{' '}
-        <InternalLink to={`/customers/${customer.id}`}>
-          {customer.firstName} {customer.lastName}
-        </InternalLink>
-      </div>
-    );
-  });
+  const expiredLeasesElements = leases
+    .filter(lease => !lease.isActive)
+    .map(({ startDate, endDate, customer }, i) => {
+      return (
+        <div key={i}>
+          <Text>{`${formatDate(startDate, i18n.language)} - ${formatDate(
+            endDate,
+            i18n.language
+          )}`}</Text>{' '}
+          <InternalLink to={`/customers/${customer.id}`}>
+            {customer.firstName} {customer.lastName}
+          </InternalLink>
+        </div>
+      );
+    });
 
-  const isNotNull = (property: boolean | null): property is boolean =>
-    property !== null;
   const getColor = (property: boolean) => (property ? 'standard' : 'secondary');
 
   return (
     <div className={styles.berthDetails}>
       <div className={styles.berthProperties}>
-        {isNotNull(gate) && (
```

Comment:
Now I remember why we had `isNotNull` function, it was because that we wanted to show Icons greyed-out if they have `false` values but remove them entirely if they are `null/undefined`.
By removing it, we only show supported the pier's properties.

Reply:
Which of these is the current wanted behavior?
A) show properties with falsy (false, null, undefined) values as gray
B) hide properties if value is null or undefined, show value as gray if false

Reply:
I'd say B

Reply:
Done.